swap_meet/vendor.py

```diff
@@ -1,2 +1,146 @@
+# simon? -- or whoever may be grading this project, please disregard
+# this commit I am working on the projects over the break to do 
+#corrections pls lmk if I should wait until after already graded that
+# I can go over them thank you!
+
 class Vendor:
-    pass
\ No newline at end of file
+    
+# Wave 01
+    def __init__(self, inventory = []):
+        self.inventory = inventory
+    
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+
+        return False
+
+
+# Wave 02
+    def get_by_id(self, id):
+        for item in self.inventory:
+            if item.id == id:
+                return item
+        return None
+
+
+
+# Wave 03 
+    def swap_items(self, other_vendor, my_item, their_item):
+        #
+        if my_item in self.inventory and their_item in other_vendor.inventory:
+            self.inventory.remove(my_item)
+            other_vendor.inventory.append(my_item)
+            other_vendor.inventory.remove(their_item)
+            self.inventory.append(their_item)
```

Comment:
This is a good opportunity to use the instance methods you defined earlier! Consider this code:

```python
            self.remove(my_item)
            other_vendor.add(my_item)
            other_vendor.remove(their_item)
            self.add(their_item)
```